config/locales/nl.yml

```diff
@@ -1,25 +1,25 @@
 nl:
   edit: Wijzig
-  failure_when_forbidden:
-  feed_latest: RubyGems.org | Recente Gems
+  failure_when_forbidden: Controleer het webadres of probeer het opnieuw.
```

Comment:
Maybe this can be: "Controleer het webadres, en probeer het opnieuw".

Reply:
Should be fixed now.